rubrics/tool/src/main/frontend/webcomponents/sakai-rubric.js

```diff
@@ -192,8 +192,10 @@ export class SakaiRubric extends SakaiElement {
       this.dispatchEvent(new SharingChangeEvent());
       this.handleShareLink();
       var sakaiItemDelete = this.querySelector("sakai-item-delete");
-      sakaiItemDelete.requestUpdate("item", this.rubric);
-      sakaiItemDelete.requestUpdate("rubric", this.rubric);
+      if(sakaiItemDelete != null) {
```

Comment:
Small things ...

Can you leave a space after the if. Also, I'd just go "if (sakaiItemDelete) {", using the falsyness.